xml/Microsoft.Vsa/IVsaDTEngine.xml

```diff
@@ -47,7 +47,7 @@
         <Parameter Name="isAttach" Type="System.Boolean" />
       </Parameters>
       <Docs>
-        <param name="isAttach">To be added.</param>
+        <param name="isAttach">True if debugger is attachable, false otherwise</param>
```

Comment:
```suggestion
        <param name="isAttach">
          <see langword="true" /> to indicate the debugger is attachable; otherwise, <see langword="false" />.</param>
```